### PR TITLE
Check authorization

### PR DESCRIPTION
**Pull request by zzuutt**
Add authorization control
if the user is authorized to view this module

### CODE REVIEW

File: Loop/DealerLoop.php [hunk 1]
```diff
@@ -217,12 +217,6 @@ public function buildModelCriteria()
                 case 'date-reverse' :
                     $query->orderByCreatedAt(Criteria::DESC);
                     break;
-                case 'title' :
-                    $query->useDealerI18nQuery()->orderByTitle()->endUse();
-                    break;
-                case 'title-reverse' :
-                    $query->useDealerI18nQuery()->orderByTitle(Criteria::DESC)->endUse();
-                    break;
```
Comment (Soldras) [inline Loop/DealerLoop.php]: Why do you remove Title order on loop ?

**Comment by zzuutt** [comment]
ha s...
it's my file not updated
I am going to correct

**Comment by Soldras** [comment]
Now you just have to squash your commit 👍

**Comment by zzuutt** [comment]
grr reprobleme avec github windows
il n'attends pas que sauvegarde le fichier ou je mets le squash

**Pull request closed without merging**